jekyll/_cci2/server/v4.1/overview/circleci-server-overview.adoc

```diff
@@ -0,0 +1,389 @@
+---
+contentTags:
+  platform:
+    - Server v4.1
+    - Server Admin
+---
+= CircleCI server v{serverversion41} overview
```

Comment:
One last thing - not sure how this works but this shows up wonky for me

![image](https://user-images.githubusercontent.com/36839689/227302649-608aed85-bbf0-4f73-a422-6fe637c85943.png)